lib/Renard/Curie/App.pm

```diff
@@ -46,6 +46,42 @@ sub setup_window {
 	$self->builder->connect_signals;
 }
 
+sub setup_keybindings {
+	my ($self) = @_;
+
+	$self->builder->get_object('main-window')->
+		signal_connect( key_press_event => \&key_pressed, $self );
+}
+
+sub key_pressed {
+	my ($window, $event, $self) = @_;
+
+	if($event->keyval == Gtk3::Gdk::KEY_Page_Down){
+		$self->page_document_component->set_current_page_forward;
+	} elsif($event->keyval == Gtk3::Gdk::KEY_Page_Up){
+		$self->page_document_component->set_current_page_back;
+	} elsif($event->keyval == Gtk3::Gdk::KEY_Up){
+		my $current = $self->page_document_component->scrolled_window->get_vadjustment;
+		my $adjustment = $current->get_value - $current->get_step_increment;
+		$current->set_value($adjustment);
```

Comment:
This code could be better as a method in the PageDrawingArea object.

e.g., `scroll_up_increment`, `scroll_down_increment`, `scroll_right_increment`, `scroll_left_increment`